src/Tribe/Views/V2/Widgets/Widget_Abstract.php

```diff
@@ -289,4 +350,78 @@ protected function args_to_context( array $arguments, Context $context ) {
 
 		return $context_args;
 	}
+
+	/**
```

Comment:
:no_entry_sign: Doc comment for parameter "$arguments" missing

<sub>hash: 88691ed3c802d2edbd80198735e0f8d1</sub>